src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheck.java

```diff
@@ -311,9 +311,9 @@ public void setClosingParens(final String parens) {
 
     /**
      * Retrieves an {@link Enum Enum} type from a @{link String String}.
-     * @param <T> the enum type
      * @param enumClass the enum class
      * @param value the string representing the enum
+     * @param <T> the enum type
```

Comment:
https://github.com/checkstyle/checkstyle/issues/2810
> The preferred order, imo, should be T, 1, 2, 3 like they are defined in the method definition.

This change goes against what was described in issue.

Reply:
Oh yeah, I'll fix this one.

Reply:
Done 👍